core/modules/file/js/file.js

```diff
@@ -37,10 +37,10 @@ Backdrop.behaviors.fileButtons = {
  */
 Backdrop.behaviors.filePreviewLinks = {
   attach: function (context) {
-    $('div.form-managed-file .file a, .file-widget .file a', context).once('file-preview-link').on('click', Backdrop.file.openInNewWindow);
+    $('.file-preview-link', context).once('file-preview-link').bind('click', Backdrop.file.openInNewWindow);
```

Comment:
`bind()` is deprecated so it should not be used. See: https://api.jquery.com/bind/. Instead `on()` and `off()` should be used. It's not clear why this was changed from the original in this code.